src/containers/Tenant/Diagnostics/Overview/TableInfo/prepareTableInfo.tsx

```diff
@@ -126,10 +128,12 @@ const prepareTableGeneralInfo = (PartitionConfig: TPartitionConfig, TTLSettings?
         }
     }
 
-    generalTableInfo.push({
-        label: i18n('label.bloom-filter'),
-        value: EnableFilterByKey ? i18n('enabled') : i18n('disabled'),
-    });
+    if (EnableFilterByKey != undefined) {
```

Comment:
It's better to use helper `valueIsDefined` or at least compare strictly `!==`. In this case it seems that there is no difference, but it generally good practice to avoid possible errors further